create-fusion-app/bin/cli.js

```diff
@@ -32,6 +32,7 @@ scaffold({
   cwd: path.join(__dirname, '..'),
   projectPath: path.join(process.cwd(), projectName),
   project: projectName,
+  packageJsonFields: {name: projectName},
```

Comment:
Minor: are we moving away from leveraging Nunjucks for templating new Fusion.js projects?

Reply:
I'm not aware of any plan. I think you're getting at why the `package.json` isn't just a Nunjucks file? Seems like it could be, but `fusion-scaffolder` already has this mechanism for replacing fields in the `package.json` so I just utilized that

Reply:
Yeah, caught me a bit by surprise given we lean on Nunjucks for our internal templates.  🤷‍♂

LGTM!